### ISSUE TITLE
Timestamps for Grants

### ISSUE THREAD

**Issue by carletex** [BuidlGuidl/grants.buidlguidl.com#38]
Let's save the timestamp (it seems that Firestore doesn't have any internal ones as I thought) for different actions.

Maybe we can track the timestamp when the grant changes to: "proposed" (apply) | "approved" | "submitted" | "completed" (rejected is not very important, but we could too).

Not sure what approach is best:

```ts
type GrantData = {
  timestamps: {
    proposed: number;
    approved: number;
    //.....
}
```

or

```ts
type GrantData = {
  proposedTimestamp:number;
  approvedTimestamp:number
  // ....
}
```

If we can query / sort them correctly... I guess it doesn't matter

**Comment by carletex** [comment]
cc @technophile-04

**Comment by technophile-04** [comment]
Umm I was thinking going with second for simplicity and maybe change name from `proposedTimestamp` => `proposedAt` 😂

Because `orderBy(fieldName)`  will be easy and fast. 

It seems firebase also supports [`orderBy(subcollection.filedName)`](https://stackoverflow.com/a/46841561) (since in first way we need to create `timestamps` subcollection). 

But yeah I think going with just fieldName will be easy, will draft a PR and we could decide if need to other way 🙌

**Comment by carletex** [comment]
Sounds great @technophile-04

**Comment by Pabl0cks** [comment]
Merged in #45
